packages/presentation/lib/screen/login/login_bloc.dart

```diff
@@ -78,36 +107,81 @@ class _LoginBloc extends BlocImpl<BaseArguments, LoginData>
   }
 
   @override
-  Future<void> auth() async {
+  String? validateLogin() {
+    if (loginValidation == ValidationErrorType.requiredErrorType) {
+      return SM.current.loginFieldRequired;
+    } else if (loginValidation == ValidationErrorType.minLengthErrorType) {
+      return SM.current.loginFieldInvalid;
+    } else {
+      null;
+    }
+  }
+
+  @override
+  String? validatePassword() {
+    if (passwordValidation == ValidationErrorType.requiredErrorType) {
+      return SM.current.passwordFieldRequired;
+    } else if (passwordValidation == ValidationErrorType.regexErrorType) {
+      return SM.current.passwordFieldInvalid;
+    } else {
+      null;
+    }
+  }
+
+  @override
+  void onChangedLogin(String changeLogin) {
+    _loginScreenFormKey.currentState?.validate();
+    loginValidation = null;
```

Comment:
_loginScreenFormKey.currentState?.validate() calls validate methods in form.
for onChangeLogin next flow: 
1. validator: (_) => bloc.validateLogin() which get loginValidation and converts it to string
if bloc.validateLogin() return null, error message will disappear 
if string => error message displayed.
2. u set loginValidation = null;
Seems it will not work correctly. only after second onChangedLogin error message will disappear

Reply:
it work correct